docs/source/user-guide/configs.md

```diff
@@ -58,6 +58,7 @@ Environment variables are read during `SessionConfig` initialisation so they mus
 | datafusion.execution.parquet.reorder_filters                            | false                     | (reading) If true, filter expressions evaluated during the parquet decoding operation will be reordered heuristically to minimize the cost of evaluation. If false, the filters are applied in the same order as written in the query                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
I recommend we add a test to https://github.com/apache/datafusion/blob/df0d9663b55068f48fb1b3234da8dea7218c805b/datafusion/sqllogictest/test_files/parquet.slt#L174-L190 as well to show this working in the SQL layer